README.md

```diff
@@ -1,36 +1,26 @@
-# unleash-proxy-client-swift
+# Unleash Swift SDK
 
-The unleash-proxy-client-swift makes it easy for native applications and other swift platforms to connect to the unleash proxy. The proxy will evaluate a feature toggle for a given [context](https://docs.getunleash.io/docs/user_guide/unleash_context) and return a list of feature flags relevant for the provided context.
+Unleash is a private, secure, and scalable [feature management platform](https://www.getunleash.io/) built to reduce the risk of releasing new features and accelerate software development. This client-side Swift SDK helps you integrate with Unleash and evaluate feature flags inside your application.
 
-The unleash-proxy-client-swift will then cache these toggles in a map in memory and refresh the configuration at a configurable interval, making queries against the toggle configuration extremely fast.
+You can use this client with [Unleash Enterprise](https://www.getunleash.io/pricing?utm_source=readme&utm_medium=swift) or [Unleash Open Source](https://github.com/Unleash/unleash).
 
-## Requirements
-
-- MacOS: 12.15
```

Comment:
Is this the correct version requirement?

Reply:
We need to check with @FredrikOseberg because he added it back in 2021. In the code I keep seeing 10.15 for macOS